admin_guide/advanced_scheduling/pod_affinity.adoc

```diff
@@ -0,0 +1,351 @@
+[[admin-guide-sched-pod-affinity]]
+= Using Pod Affinity and Anti-Affinity
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+_Pod affinity_ and _pod anti-affinity_ allow you to specify rules about how pods should be placed relative to one another. For example, you could spread or pack pods within a service or relative to pods in other services or, you can make sure a pod is scheduled on a specific type of machine.
```

Comment:
@wmengRH 
I am using this document. 
https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#per-pod-configurable-eviction-behavior-when-there-are-node-problems-alpha-feature

Please tell me if i am incorrect, which i probably am wrong!